src/scripts/main.js

```diff
@@ -1,3 +1,254 @@
 'use strict';
 
 // write your code here
+
+const button = document.querySelector('button');
```

Comment:
whicch exact button you're getting here? Maybe use class to select exact button?